packages/react-scripts/template/src/Toast.css

```diff
@@ -0,0 +1,39 @@
+.Toast {
```

Comment:
Might be worth name spacing this, if there is any expectation the toast will appear in production?

`.cra-Toast` maybe ?